README.md

```diff
@@ -7,8 +7,16 @@ nlp_primitives is a Python library with Natural Language Processing Primitives,
 nlp_primitives allows you to make use of text data in your machine learning pipeline in the same pipeline as the rest of your data.
 
 ### Install
+There are two options for installing nlp_primitives. Both of the options will also install Featuretools, if it is not already installed.
+
+The first option is to install a version of nlp_primitives that does not include Tensorflow. With this option, primitives that depend on Tensorflow cannot be used. To install nlp_primitives without Tensorflow run:
```

Comment:
I think for now, since only USE primitive depends on Tensorflow, we should mention it individually so the user knows its only one primitive they lose out on by skipping the Tensorflow install

Reply:
Updated